src/extension/background-script/connectors/commando.ts

```diff
@@ -142,7 +142,15 @@ export default class Commando implements Connector {
   }
 
   get supportedMethods() {
-    return supportedMethods;
+    return [
+      "getInfo",
+      "keysend",
+      "makeInvoice",
+      "sendPayment",
+      "signMessage",
+      "getBalance",
+      ...supportedMethods,
```

Comment:
Not happy about the API surface here since some of the methods are in `webln` and others are only accessible via `webln.request()` 🤔

Reply:
I agree. It is two separate APIs right? (the WebLN high level API that is the same across all node types, and then the WebLN.request low level API which are methods exposed directly from the node).